data/ships.txt

```diff
@@ -317,12 +317,12 @@ ship "Bastion"
 		"hull" 4200
 		"required crew" 32
 		"bunks" 47
-		"mass" 580
+		"mass" 540
```

Comment:
Why are you changing an existing ship?

Reply:
Because the Bastion used to be heavier than e.g. the Vanguard warship, and I think it ought to remain a medium warship, therefore I believe slightly removing some unnecessary mass is justified here. (With this change the Bastion's free outfit space becomes 14/450, comparable to other medium warships.)

Reply:
Uh, mass is independent of outfit space...